forge-gui/res/cardsfolder/upcoming/distinguished_conjurer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Distinguished Conjurer
+ManaCost:1 W
+Types:Creature Human Wizard
+PT:1/2
+T:Mode$ ChangesZone | ValidCard$ Creature.Other+YouCtrl | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigGainLife | TriggerDescription$ Whenever another creature enters the battlefield under your control, you gain 1 life.
+SVar:TrigGainLife:DB$ GainLife | LifeAmount$ 1
+A:AB$ ChangeZone | Cost$ 4 W T | ValidTgts$ Creature.Other+YouCtrl | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select another target creature you control | RememberTargets$ True | SubAbility$ DBReturn | SpellDescription$ Exile another target creature you control, then return it to the battlefield under its owner's control.
```

Comment:
hmm RememberTargets instead of RememberChanged seems wrong

Reply:
You sure ? I copied it from another card and during tests it worked

Reply:
I just said it "seems" wrong... I'm not 100% sure without looking into it

RememberTargeted remembers the target whether the card changes zone or not
RememberChanged only remember the card if it actually changes zone

Reply:
No worries. thanks for the clarification. It seems that both will work functionally but RememberChanged  might be the more safe option rules wise